docs/_docs/filename.md

```diff
@@ -79,6 +79,9 @@ Note that here, each slash (`/`) corresponds to a different directory:
 * `count`: counts the current image downloaded during a batch download. Useful to have unique counters per batchs.
 * `num`: counter that goes up for every file with the same name. Useful to prevent overwriting already existing files.
 * `position`: the position of the image in the results list or gallery (starts at 1).
+* ~~`directory`: when renaming, store the local path for the current file (useful to rename images, without changing directory structure).~~ Beware this token is replaced by old_directory but seems to still work in the origin filename field, but not on the destination filename field.
```

Comment:
This token does not exist anymore.

What you mean by "still work in the origin filename field" can actually be extended to pretty much anything. When renaming, anything that looks like a token (even if it doesn't exist, like `%toto%` or, as is the case here, `%directory%`) will be treated as a `*` when matching files (or to be precise the `.+?` regex).

Reply:
To add to this, I guess the way the renamer works should be expanded in the docs, because it's indeed quite obscure.

Reply:
Ok since it's not a token anymore I'll remove it from the changes.